src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+                {
+                    NeedInitalizeLayouter = false;
+                    return;
+                }
+
+                if ((_itemsSource != null) && (_itemsTemplate != null))
+                {
+                    if (InternalItemSource == null)
+                    {
+                        InternalItemSource = ItemsSourceFactory.Create(this, _itemsLayouter);
+                    }
+
+                    _itemsLayouter.Initialize(this);
+                    _itemsLayouter.RequestLayout(0.0f, true);
+                    if (ScrollingDirection == Direction.Horizontal)
+                    {
+                        ContentContainer.SizeWidth =
+                            _itemsLayouter.CalculateLayoutOrientationSize();
+                    }
+                    else
+                    {
+                        ContentContainer.SizeHeight =
+                            _itemsLayouter.CalculateLayoutOrientationSize();
+                    }
+                    NeedInitalizeLayouter = false;
+                }
+                else NeedInitalizeLayouter = true;
+            }
+        }
+
+        /// <summary>
+        ///  DataTemplate of group header. Group feature is not supported yet.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public DataTemplate GroupHeaderTemplate;
+
+        /// <summary>
+        /// DataTemplate of group footer. Group feature is not supported yet.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DataTemplate GroupFooterTemplate; 
+
+        /// <summary>
+        /// Binding Property of selected item in single selection.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public static readonly BindableProperty SelectedItemProperty =
+			BindableProperty.Create(nameof(SelectedItem), typeof(object), typeof(CollectionView), null,
+				propertyChanged: (bindable, oldValue, newValue)=>
+                {
+			        var colView = (CollectionView)bindable;
+                    var args = new SelectionChangedEventArgs(oldValue, newValue);
+                    oldValue = colView.selectedItem;
+                    
+                    foreach(View item in colView.ContentContainer.Children)
+                    {
+                        if (item as ViewItem)
+                        {
+                            var vItem = item as ViewItem;
+                            if (vItem.BindingContext == null) continue;
```

Comment:
Linq와 `is` 키워드를 사용하시면 간단하게 줄일 수 있습니다.
Linq를 사용하려면 `using System.Linq;`를 선언해야합니다.
(C#에는 한줄 80 글자수 제한은 없습니다.)
```suggestion
                    foreach (ViewItem item in colView.ContentContainer.Children.Where((item) => item is ViewItem))
                    {
                        if (item.BindingContext == null) continue;
```